test/evp_libctx_test.c

```diff
@@ -71,6 +71,42 @@ static const char *getname(int id)
 }
 #endif
 
+static int test_evp_cipher_api_safety(void)
+{
+    int ret = 0;
+    EVP_CIPHER_CTX *ctx = NULL;
+
+    ctx = EVP_CIPHER_CTX_new();
+
+    if (!TEST_ptr(ctx)) {
+        TEST_info("Unable to allocate cipher context");
```

Comment:
Not a big fan of using TEST_info() everywhere.. The TEST_macros should be sufficient.

Reply:
I rather like the additional clarity of the infomational printouts, but ok.